components/style/values/computed/image.rs

```diff
@@ -54,6 +54,8 @@ pub enum LineDirection {
     Angle(Angle),
     /// A corner.
     Corner(X, Y),
+    /// A Position and an Angle for legacy `-moz-` prefixed gradient.
+    PositionAngle(Option<Position>, Option<Angle>),
```

Comment:
Call that `MozPosition` and put `#[cfg(feature = "gecko")]` on it and on every code that uses it.